src/HotChocolate/Core/test/Execution.Tests/Errors/ErrorHandlerTests.cs

```diff
@@ -58,6 +58,34 @@ public async Task AddClassErrorFilter()
                     .AddErrorFilter<DummyErrorFilter>());
         }
 
+        [Fact]
+        public async Task AddClassErrorFilter_SchemaBuiltViaServiceExtensions_ErrorFilterWorks()
+        {
+            var serviceCollection = new ServiceCollection();
+            var schema = await serviceCollection
+                .AddGraphQLServer()
+                .AddErrorFilter<DummyErrorFilter>()
+                .AddQueryType<Query>()
+                .BuildSchemaAsync();
```

Comment:
Why do you not sue BuildRequestExecutorAsync here?

Reply:
Didnt know about that method, fixed it.